utils/import_current_votes.py

```diff
@@ -5,12 +5,19 @@
 from brownie import accounts
 from brownie.network.transaction import TransactionReceipt
 from utils.test.helpers import ETH
-from utils.config import LDO_HOLDER_ADDRESS_FOR_TESTS, get_is_live, contracts
+from utils.config import LDO_HOLDER_ADDRESS_FOR_TESTS, get_is_live, contracts, network_name
+
+
+def _get_network():
```

Comment:
After discussion we decided to keep current convention, the vote script will  be moved to the root folder, those changes do not needed.